uwb-beacon-firmware/src/uavcan/topics_publisher.cpp

```diff
@@ -106,6 +119,10 @@ void topics_publisher_start(Node& node)
     res = tag_pos_pub->init();
     chDbgAssert(res == 0, "publisher failed");
 
+    data_packets_pub.construct<Node&>(node);
+    res = data_packets_pub->init();
+    chDbgAssert(res == 0, "publisher failed");
```

Comment:
maybe worth mentioning the name of the publisher that failed to init in the error message